WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/logic/ui/listeners/IntelliJListener.java

```diff
@@ -68,6 +65,6 @@ public void removeListeners() {
         connection.disconnect();
         activityListener.removeListeners();
         parent.dispose(); // removing EditorFactoryListener
```

Comment:
Yes, that is actually a dummy disposable parent and that's apparently the proper way to remove this listener (although I'm not sure how fast does this happen). The next line is causing a warning, because `removeEditorFactoryListener` has been deprecated. I suggest to have both for now, just to make sure that the listeners are removed for sure - deprecated still doesn't mean deleted.